### ISSUE TITLE
[SUGGESTION] Limit human self redeems

### ISSUE THREAD

**Issue by mikan1216** [Azurblau/Zombies.Zone#117]
It's great that you can redeem for second chance. But because many people use it so much, map doesn't progress any, because no more zombies. 

Can we limit to for example, once in one map.

**Comment by WardenPotato** [comment]
+1

**Comment by Aquid0** [comment]
+!

**Comment by TheFray89** [comment]
especially for Objective maps.
+1

**Comment by Loickun** [comment]
Nah, you cant redeem when there are alot of players and when there are less than 25 players its just more of a buff for zmainers. Besides there is a cool down for it.

**Comment by Loickun** [comment]
Do something against zmainers first. 
Maybe only redeem between waves but there for disable boss for Wave 2.
WHEN there are less than 25 players online.
